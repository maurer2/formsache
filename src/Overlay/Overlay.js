import React from 'react';
import PropTypes from 'prop-types';
import style from './Overlay.module.scss';

const Overlay = ({ message }) => (
  <div className={style.overlay}>
    { message }
  </div>
);

export default Overlay;

Overlay.propTypes = {
  message: PropTypes.string.isRequired,
};
