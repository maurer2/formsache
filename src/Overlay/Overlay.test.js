import React from 'react';
import { shallow } from 'enzyme';
import Overlay from './Overlay';

describe('Overlay', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Overlay
      message="Mewo Meow"
    />);
  });

  test('snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
