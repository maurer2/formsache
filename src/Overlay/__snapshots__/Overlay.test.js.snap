// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay snapshot 1`] = `
<div
  className="overlay"
>
  Mewo Meow
</div>
`;
